tracing-log/tests/init_interests_cache_before_global_subscriber.rs

```diff
@@ -0,0 +1,101 @@
+#![cfg(feature = "interest-cache")]
```

Comment:
This test isn't run as part of the CI. Should I add it to the CI config?

Reply:
Yes, there should be a CI job that runs with `--all-features` for every crate.